docs/site/Model.md

```diff
@@ -140,7 +140,7 @@ LB4 supports creating a model that allows both well-defined but also arbitrary
 extra properties for **NoSQL** databases such as MongoDB. You need to disable
 `strict` mode in model settings. Besides modifying model settings directly, it
 can also be done through the CLI by setting `allowed free-from properties` to
```

Comment:
I know this isn't part of your PR but can you fix `allowed free-from properties` to `allow free-form properties`?

Reply:
Good catch! Changed it to `Allow additional (free-form) properties` to match with the current CLI.